### PR TITLE
Survey

### PR DESCRIPTION
**Pull request by JustoRago**

### CODE REVIEW

**Review by JustoRago — COMMENTED** [review]
While this is technically functional and can pass the automatic test file, it is by no means complete code. It lacks all styling and formatting and relies entirely on linebreak elements.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<html>
+<h1 id="title">Build a Survey</h1>
+<p id="description">We are building a survey.</p>
+<form id="survey-form">
+  <label id="name-label">Name</br>
+  <input id="name" type="text" placeholder="Enter your name" required></label>
+</br><label id="email-label">Email</br><input id="email" placeholder="Enter your Email" type="email" required></label>
+  </br><label id="number-label">Age (optional)</br>
+  <input id="number" type="number" placeholder="Age" min="10" max="99"></label>
+  </br><select id="dropdown"><option></option><option></option></select>
```
Comment (JustoRago) [inline index.html:11]: I belive that the options in the dropdown should not be empty :smile:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<h1 id="title">Build a Survey</h1>
```
Comment (JustoRago) [inline index.html:3]: In order to comply with html/css standards, there should be head and body elements, with the head containing elements such as the title, character encoding (meta element) and a link to a css stylesheet. 

Using these will give you many new possibilities!:rocket:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<p id="description">We are building a survey.</p>
+<form id="survey-form">
+  <label id="name-label">Name</br>
```
Comment (JustoRago) [inline index.html:6]: The linebreak elements would be unnecessary here with css-based styling, which would make things easier on you.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  <input id="number" type="number" placeholder="Age" min="10" max="99"></label>
+  </br><select id="dropdown"><option></option><option></option></select>
+  </br>
+  <legend>One-Two </br>
+<label><input id="one" type="radio" name="one-two" value="one">One</label>
+<label><input id="two" type="radio" name="one-two" value="two">Two</label></legend>
+<legend>Multiple-choice</br>
+<label><input id="a" type="checkbox" name="multiple-choice" value="a">A</label>
+<label><input id="b" type="checkbox" name="multiple-choice" value="b">B</label>
+<label><input id="c" type="checkbox" name="multiple-choice" value="c">C</label></legend>
```
Comment (JustoRago) [inline index.html:19]: I think a class selector should be used here instead of multiple id selectors, so that the options can be more easily styled.

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @asdt560 , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- Great use of GitHub review --inline comments features

## Required Changes ♻️

- [x] Good job so far 👏.  While giving reviews, **please try to be polite.** This is by phrasing your sentences in a way that is `suggesting` or `recommending` or requesting instead of giving orders. This makes the code author feel like he/she is not being undermined or being forced to change his/her own codes but rather feels like it is a way for improvement 😉
```
Examples: 
This is a polite suggestion: "I think it would be better to use semantic tags"
This reads as an order: "Use semantic tags"
```

- [x] Also, try to give a detailed explanation about **WHAT you are suggesting and WHY you are suggesting it. This helps the code author to clearly understand what changes to be implemented 😃 👍 .** 💡 You may also consider **giving some hints or references** that may help him/her understand more what to be done 😉


## Optional suggestions

- [x] [OPTIONAL] Please try to use emojis and/or GiFs in your reviews because they make them look more human-friendly 🤖 👽 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

⚠️ **NOTE** In order to implement suggestions provided, please edit the comments you have already left while performing a review 😎 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @asdt560 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights
 - Great use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**